src/main/java/duke/command/AException.java

```diff
@@ -0,0 +1,66 @@
+package duke.command;
+
+/**
+ * Represents every exception that Duke may encounter
+ * when the user uses Duke.
+ */
+public class AException {
```

Comment:
You may consider passing a `String` that describes the error when you construct an `AException` and create an object for every exception that you encounter. Hence, every error is described by one exception object instead of just calling a function that directly prints to stdout.